Adding support for more than 10 slides when using the --imageexport option

calfonso opened the pull request:
Adding support for more than 10 slides when using the --imageexport option

```
* Using a hex string now instead of a decimal for the exported png image name,
  which ensures the joining of the images is done in the right order.  If you use
  a decimal to name the png files, the order will be 0,10,11,12,1,2,3..  Using hex
  eliminates this problem.
```

abourget commented:
Actually, wouldn't it be better to use %05d ? hex(2) produces '0x1' and hex(16) produced '0x10'.

Or we could use hex as "%06x" ?

calfonso commented:
Yeah, I like the approach of just adding the leading 0's.  I'll change the patch and rewrite history.

calfonso committed:
Changed the PNG file naming when using -imageexport to support many layers.

* When combinging the PNG files into a pdf, we are relying on the file names
  for ordering of the slides.  Since there were no leading zeros, we would run
  into an issue with slides joining out of order.
